spec/graphql/resolvers/customer_portal/customers/usage_resolver_spec.rb

```diff
@@ -125,47 +131,56 @@
   it_behaves_like "requires a customer portal user"
 
   it "returns the usage for the customer" do
-    Subscriptions::ChargeCacheService.expire_for_subscription(subscription)
-    result = execute_graphql(
-      customer_portal_user: customer,
-      query:,
-      variables: {
-        subscriptionId: subscription.id
-      }
-    )
+    travel_to(now) do
+      Subscriptions::ChargeCacheService.expire_for_subscription(subscription)
+      result = execute_graphql(
+        customer_portal_user: customer,
+        query:,
+        variables: {
+          subscriptionId: subscription.id
+        }
+      )
 
-    usage_response = result["data"]["customerPortalCustomerUsage"]
+      usage_response = result["data"]["customerPortalCustomerUsage"]
 
-    aggregate_failures do
-      expect(usage_response["fromDatetime"]).to eq(now.beginning_of_month.iso8601)
-      expect(usage_response["toDatetime"]).to eq(now.end_of_month.iso8601)
-      expect(usage_response["currency"]).to eq("EUR")
-      expect(usage_response["issuingDate"]).to eq(now.to_date.end_of_month.iso8601)
-      expect(usage_response["amountCents"]).to eq("405")
-      expect(usage_response["totalAmountCents"]).to eq("405")
-      expect(usage_response["taxesAmountCents"]).to eq("0")
+      aggregate_failures do
```

Comment:
when editing tests, don't hesitate to remove the `aggregate_failures do ... end`. We have enabled it globally since.
It's fine not to do it, just sayin' 😛